rs/types/types/src/consensus/dkg.rs

```diff
@@ -375,12 +380,55 @@ fn build_tagged_transcripts_map(
                 .ok_or_else(|| ProxyDecodeError::MissingField("TaggedNiDkgTranscript::transcript"))
                 .and_then(|t| {
                     Ok((
-                        NiDkgTag::try_from(tagged_transcript.tag).map_err(|e| {
-                            ProxyDecodeError::Other(format!(
-                                "Failed to convert NiDkgTag of transcript: {:?}",
-                                e
-                            ))
-                        })?,
+                        /////////////////////////////////////////////////
+                        // TODO: ideally we extend existing tests where build_tagged_transcripts_map
```

Comment:
It could be that there are no specific unit tests for the (de)serialization of general summaries, and that this is only tested by system/integration tests. For IDKG we have `test_summary_proto_conversion`, probably it makes sense to add something similar for vet KD once that is possible.

It will also be tested as part of the `cup_compatibility_test` but only once we include the new variant in the `ExhaustiveSet`.

Reply:
I see, thanks! Given that this new behaviour will eventually be tested as part of the `cup_compatibility_test`, I simply removed my TODO here.

Please let me know if you want me to create any (Consensus) tickets for adding an NI-DKG-specific `test_summary_proto_conversion` once that is possible or if I shall add a TODO with a specific ticket number.

Reply:
I extended [CON-1413](https://dfinity.atlassian.net/browse/CON-1413) to mention that we should test the conversion once we can create these kinds of payloads.